homeassistant/helpers/device_registry.py

```diff
@@ -49,6 +50,12 @@ class _DeviceIndex(NamedTuple):
     connections: dict[tuple[str, str], str]
 
 
+class DeviceEntryType(Enum):
```

Comment:
```suggestion
class DeviceEntryType(str, Enum):
```

That make it compatible will converter like json